avocado/utils/process.py

```diff
@@ -538,6 +538,13 @@ def stop(self):
             self.terminate()
         return self.wait()
 
+    def get_pid(self):
+        """
+        Reports PID of this process
+        """
+        self._init_subprocess()
```

Comment:
I had a weird feeling about this lazy (and implicit) init of the subprocess, but since all other `get_*()` methods do it, it's the right way to go.